CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.4.1
+
+This is a maintenance release.
+
+* [TK-270](https://tickets.puppetlabs.com/TK-270) Restore the definition of
```

Comment:
I would maybe say "fix a bug that prevented the use of the 1-arity ..."; not too picky on the wording other than I'd probably be explicit in calling out that it was a bugfix.